osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -141,6 +143,23 @@ private void load(Storage storage, FrameworkConfigManager frameworkConfig)
             });
         }
 
+        private void readStableConfig()
```

Comment:
I still don't quite understand why this has to be here. Can this not be done inside of `FileBasedIPC`?

Reply:
The reason it's being read here is because it's being it's being put in the dependency container after it was read.
`dependencies.Cache(stableInfo);` 

 Moving the read to `FileBasedIPC` and losing DI on it means that `SetupScreen` can't use this binding any more to refresh the values on screen:

```
stableInfo.StablePath.BindValueChanged(_ =>
{
    Schedule(reload);
});
```

and `StablePathSelectScreen` then needs to access the value of `stableInfo` through `FileBasedIPC`